scripts/test.sh

```diff
@@ -2,4 +2,4 @@
 
 GO111MODULE="on" go test -race -mod=vendor ./commands/... ./do/... ./pkg/... .
 # disable until integration tests green on windows
-#GO111MODULE="on" go test -race -mod=vendor ./integration
```

Comment:
Rather than uncomment line, just delete this file and use `make test` 😄 

Update `.travis.yml` accordingly

Reply:
👯‍♂